csrankings.csv

```diff
@@ -13672,7 +13677,8 @@ Nihan K. Çiçekli,Middle East Technical University,http://user.ceng.metu.edu.tr
 Nihan Kesim Cicekli,Middle East Technical University,http://user.ceng.metu.edu.tr/~nihan,NOSCHOLARPAGE
 Nihar B. Shah,Carnegie Mellon University,http://www.cs.cmu.edu/~nihars,BF39lMQAAAAJ
 Nihat Altiparmak,University of Louisville,http://cecs.louisville.edu/nihat,HvPsovYAAAAJ
-Nika Haghtalab,Cornell University,http://www.cs.cornell.edu/~nika,C6pnolkAAAAJ
+Nika Haghtalab,Cornell University,http://www.cs.cornell.edu/~nika,C6pnolkAAAAJ
+Nikhil Srivastava,University of California - Berkeley,https://math.berkeley.edu/~nikhil/,6ReT2voAAAAJ

```

Comment:
No sign of cross-appointment. https://www2.eecs.berkeley.edu/Faculty/Lists/faculty.html#S